dataactcore/migrations/versions/a62138efd429_add_tas_id_to_sf133.py

```diff
@@ -0,0 +1,42 @@
+"""Add tas_id to SF133
+
+Revision ID: a62138efd429
+Revises: d7476f20b59f
+Create Date: 2016-11-22 20:14:06.653535
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'a62138efd429'
+down_revision = 'd7476f20b59f'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+
+
+def upgrade_data_broker():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('sf_133', sa.Column('tas_id', sa.Integer(), nullable=True))
+    op.create_foreign_key('fk_tas', 'sf_133', 'tas_lookup', ['tas_id'], ['tas_id'])
```

Comment:
I _think_ postgres scopes FK's to their table, but this is good advice. Will implement, thanks!